index.html

```diff
@@ -12,11 +12,11 @@
     </script>
     <script src="https://ucarecdn.com/widget/2.5.5/uploadcare/uploadcare.full.min.js"></script>
     <script src="https://cdn.rawgit.com/zenorocha/clipboard.js/v1.5.12/dist/clipboard.min.js"></script>
-    <script src="vendor/aframe-master.min.js"></script>
+    <script src="vendor/aframe.min.js"></script>
```

Comment:
I think we can remove the A-Frame on the vendor folder and use the CDN builds: https://aframe.io/releases/1.3.0/aframe.min.js

We were probably using a custom build of in-between releases.

Reply:
Gotcha, I kinda suspected as much. Will change